subgraph-js/src/__tests__/buildSubgraphSchema.test.ts

```diff
@@ -80,7 +87,6 @@ type Money {
       }
     }`;
     const schema = buildSubgraphSchema(gql`
-      "Description text on 'SchemaDefinition' nodes supported as per the October 2021 Edition of the spec."
```

Comment:
**question** Is supporting this no longer needed?

Reply:
oh, i don't know why i dropped that? fixed.